### ISSUE TITLE
Client::loadFlags() triggers type error when request in localFlags() doesn't return data

### ISSUE THREAD

**Issue by mdio** [PostHog/posthog-php#50]
If the API request in [localFlags()](https://github.com/PostHog/posthog-php/blob/master/lib/Client.php#L386) returns null, [loadFlags()](https://github.com/PostHog/posthog-php/blob/master/lib/Client.php#L386) causes this type error in PHP 8.0:
```
array_key_exists(): Argument #2 ($array) must be of type array, null given
```

**Comment by joelwmale** [comment]
Yep, happens when the server times out, or there is no internet connection, happening on 8.1 as well.
